makefiles/Makefile.win.mk

```diff
@@ -21,6 +21,10 @@ STATIC_PRE_LIB = $(OR_ROOT)lib\\
 STATIC_POST_LIB = .lib
 O=obj
 E=.exe
+L=.lib
```

Comment:
**nix* is not in my purview right at the moment. I am focused on Windows and Windows only right now. Would a separate commit work?